ext/openssl/ossl_ssl.c

```diff
@@ -2598,6 +2601,11 @@ Init_ossl_ssl(void)
      */
     mSSLExtConfig = rb_define_module_under(mOSSL, "ExtConfig");
 
+#ifdef OPENSSL_NO_SOCK
+    rb_define_const(mSSLExtConfig, "OPENSSL_NO_SOCK", Qfalse);
+#else
+    rb_define_const(mSSLExtConfig, "OPENSSL_NO_SOCK", Qtrue);
+
```

Comment:
mSSLExtConfig is meant for internal use (by lib/*.rb). This constant isn't actually used anymore. I think I forgot to remove it in commit 998d66712a78c623977db37fc8c6f95502b71d3c.